ansible_collections/arista/avd/README.md

```diff
@@ -59,7 +63,7 @@ Ansible galaxy hosts all stable versions of this collection. Installation from a
 
 ## Ask a question
 
-Support for this `arista.avd` collection is provided by the community directly in this repository. If you have any questions, please leverage the GitHub [discussions board](https://github.com/aristanetworks/ansible-avd/discussions).
+Support for the `arista.avd` collection is provided by the community directly in this repository. If you have any questions, please leverage the GitHub [discussions board](https://github.com/aristanetworks/ansible-avd/discussions).
```

Comment:
Should we mention that if you have a support agreement you can contact Arista support?

Reply:
@ClausHolbechArista Just pushed a small note on this topic.  Please let me know if it requires any edits